### ISSUE TITLE
Example in https://github.com/0ceal0t/Dalamud-VFXEditor/wiki/Example:-Replacing-Fire-animation-with-Fell-Cleave crashes client

### ISSUE THREAD

**Issue by breadposter** [0ceal0t/Dalamud-VFXEditor#53]
Followed the example in https://github.com/0ceal0t/Dalamud-VFXEditor/wiki/Example:-Replacing-Fire-animation-with-Fell-Cleave exactly to see where some of my own attempts were going wrong. Every step works until clicking 'UPDATE' at the end, at which point the client crashes.

The Dalamud log file shows the following:

```Unhandled native exception occurred at ffxiv_dx11.exe+1B9B2A
Code: C0000005
Dump at: C:\Users\Ruby\AppData\Roaming\XIVLauncher\addon\Hooks\6.4.0.44\..\..\..\dalamud_appcrash_20220821_162632_405_13396.dmp
Time: 2022-08-21 16:26:32.6405633 GMT-7

   at System.Environment.get_StackTrace()

   at Dalamud.EntryPoint.VehCallback() in C:\goatsoft\companysecrets\dalamud\EntryPoint.cs:line 70

   at VFXEditor.Interop.ResourceLoader.ReadSqpackHandler(IntPtr pFileHandler, SeFileDescriptor* pFileDesc, Int32 priority, Boolean isSync)

   at VFXEditor.Interop.ResourceLoader.ReloadPath(String gamePath, String localPath, List`1 papIds)

   at VFXEditor.PAP.PAPDocument.Update()

   at VFXEditor.FileManager.FileManagerDocument`2.DisplayFileControls()

   at VFXEditor.PAP.PAPDocument.DrawBody()

   at VFXEditor.FileManager.FileManagerDocument`2.Draw()

   at VFXEditor.FileManager.FileManager`3.DrawBody()

   at VFXEditor.Dialogs.GenericDialog.Draw()

   at VFXEditor.Plugin.Draw()

   at Dalamud.Interface.UiBuilder.OnDraw() in C:\goatsoft\companysecrets\dalamud\Interface\UiBuilder.cs:line 473

   at Dalamud.Interface.Internal.InterfaceManager.Display() in C:\goatsoft\companysecrets\dalamud\Interface\Internal\InterfaceManager.cs:line 1088

   at ImGuiScene.RawDX11Scene.Render() in D:\a\Dalamud\Dalamud\lib\ImGuiScene\ImGuiScene\RawDX11Scene.cs:line 143

   at Dalamud.Interface.Internal.InterfaceManager.PresentDetour(IntPtr swapChain, UInt32 syncInterval, UInt32 presentFlags) in C:\goatsoft\companysecrets\dalamud\Interface\Internal\InterfaceManager.cs:line 535

   at Dalamud.Game.Framework.HandleFrameworkUpdate(IntPtr framework) in C:\goatsoft\companysecrets\dalamud\Game\Framework.cs:line 412
```

This is what my client looks like at the time of the crash:

![image](https://user-images.githubusercontent.com/92342730/185815449-2db15c7c-d891-42cb-9517-eb6b5d0fce9d.png)

I am running no other mods aside from VFXEditor at this time.

**Comment by breadposter** [comment]
After trying the same approach a few more times (saving the hkx file in a different location to see if that was an issue), sometimes the same thing happens, but sometimes the characters gets locked into a single animation after casting. 

Additionally, I think change anything in action.pap may be broken in general for me? Another thing I've tried is simply loading the Fire .pap in both inputs, then pressing update without doing anything else, and it already changes the animation despite the fact that I've not made any changes. I've attached a video to show the animations breaking: https://streamable.com/d9t17h
